### PR TITLE
Add support to split the single client file into multiple files.

### PR DESCRIPTION
**Pull request by alexdupre**
Two new options have been added:
--splitFiles to split the client code generation into multiple files
--enableFmt  to disable formatting the generated files with scalafmt

The huge gitlab schema compiled in the scripted tests has been converted
to use these new options.

### CODE REVIEW

File: docs/docs/client.html [hunk 1]
```diff
@@ -64,10 +64,12 @@
   def genView(value: Boolean): CalibanSettings                  // Provide a case class and helper method to select all fields on an object (default: false)
   def scalarMapping(mapping: (String,String)*): CalibanSettings // A mapping from GraphQL scalar types to JVM types, as unknown scalar types are represented as String by default.
   def imports(values: String*): CalibanSettings                 // A list of imports to be added to the top of a generated client
+  def splitFiles(value: Boolean): CalibanSettings               // Split single client object into multiple files (default: false)
```
Comment (ghostdogpr) [inline docs/docs/client.html:67]: Ah, to modify the docs it's not directly in the html, look at the .md files in the `vuepress` folder.

Comment (alexdupre) [reply docs/docs/client.html:67]: Fixed, thanks

**Review by blast-hardcheese — CHANGES_REQUESTED** [review]
Needs https://github.com/alexdupre/caliban/pull/1 otherwise the sbt plugin will error if `splitFiles(true)` is used.

One question as well

**Comment by ghostdogpr** [comment]
I'll review everything in #964 so let's close this one

**Pull request closed without merging**